src/components/Router/StepsRouter.js

```diff
@@ -30,7 +30,7 @@ class StepsRouter extends Component {
         <div className={classNames(theme.content,{
           [theme.fullScreenContentWrapper]: isFullScreen
         })}>
-          <CurrentComponent {...{...options, ...globalUserOptions, ...otherProps}}
+          <CurrentComponent {...{...options, ...globalUserOptions, ...otherProps, back}}
```

Comment:
This is fine, however the way it's done for `NavigationBar` is more readable, so minor but I would suggest to pass it as `back={back}`.